### DISCUSSION TITLE
Shuffle command

### DISCUSSION THREAD

**Discussion by ghost** [TannerGabriel/discord-bot#115] [Ideas]
Not really anything urgent or big, but my friends and I like to queue entire playlists off spotify and a shuffle command would be nice. Thanks for all your work btw!

**Comment by TannerGabriel** [comment]
Shuffle is already implemented and Spotify playlist can be enabled, which I could do if it would help you.

**Comment by samdammers** [comment]
Added a shuffle command in https://github.com/TannerGabriel/discord-bot/pull/139 I believe adding a playlist already does do a shuffle, but if you don't like the order you can shuffle again now
